cypress/support/index.js

```diff
@@ -1,3 +1,11 @@
 Cypress.Screenshot.defaults({
   screenshotOnRunFailure: true,
 });
+
+// eslint-disable-next-line import/prefer-default-export
+export const el = {
```

Comment:
This can live somewhere better I think, can it go in a `frontpageElements` file or something